test/schema/home/home_page_artwork_modules.js

```diff
@@ -109,6 +116,7 @@ describe('HomePageArtworkModules', () => {
           'followed_galleries',
           'saved_works',
           'current_fairs',
+          'followed_artist',
           'related_artists',
```

Comment:
Basically, I see the two as belonging to the same logical group. i.e. I’d want a client to be able to specify in their preferred order `FOLLOWED_AND_RELATED_ARTIST` and get back the 2 rails in that spot. Do you see a simple way to make that happen?